### ISSUE TITLE
Really need a way to minimize/alter build output information

### ISSUE THREAD

**Issue by Aaron-Pool** [vuejs/vue-cli#4133]
### What problem does this feature solve?
We're developing an app that's behind an enterprise authentication setup (ntlm) that doesn't play well with the yarn dev server (and yes, several people on my team have tried to get around this problem, without long term success). As a result, instead of the preferred `yarn serve`, we use `yarn build --watch` during ongoing development. Unfortunately, after every update, the build process displays every chunk that has been generated, which _completely_ pushes useful errors and warnings from the linter offscreen.

As of now, there is no way for a user to disable this build output. I've seen several requests to be able to disable/alter build logging, but it seems like the Vue-cli devs struggle to grasp a tangible example for when this would add substantial value to the user. I understand that--it is, of course, important to prioritize features based on actual value added, so I just wanted to create this issue to give a concrete example where being unable to suppress or minimize build output information _substantially_ frustrates the dev experience. Instead of being able to leave the terminal open in a separate screen and glance at it for linting errors, I have to scroll the terminal back up a full page length every time I save to see any compilation warnings or errors.

### What does the proposed API look like?
`--verbosity-level none | minimal | debug | default`

**Comment by LinusBorg** [comment]
I assume it would be enough to make this api public?

https://github.com/vuejs/vue-cli/blob/dev/packages/%40vue/cli-service/lib/commands/build/index.js#L203

**Comment by Aaron-Pool** [comment]
@LinusBorg, that should be sufficient, yes. Any output from webpack plugins would still be printed, correct? And linters?

The optimal solution in my case would just be a flag that toggles the `log(formatStats(...))` statement. But I get that beggars can't be choosers, so I'll take what I can get 😃

**Comment by LinusBorg** [comment]
If I'm not mistaken, you should be able to use `--silent` right now, it's just not public in the sense that it's not documented.

Report back if the kind of output you get with it is ok.

**Comment by Aaron-Pool** [comment]
@LinusBorg Yup, looks like what I want. 👍 I can trust this option to stick around, yeah? Even though it isn't documented?

**Comment by LinusBorg** [comment]
I think we can use this issue to discuss that we make it public. 

It won't disappear spontaneously anytime soon either way.

**Comment by Aaron-Pool** [comment]
Great, appreciate the prompt response, @LinusBorg !

**Comment by ulyssesp** [comment]
This seems to be related: `vue-cli-service serve --silent` shows all tslint warnings and errors. It would be helpful to just show the errors not tslint warnings. Have tried all sorts of `vue.config.js` options, including those that should affect the webpack-dev-server.
